Contents/Code/__init__.py

```diff
@@ -666,7 +679,7 @@ def anidbTvdbMapping(self, metadata, anidb_id, error_log):
   ### AniDB collection mapping - complement AniDB movie collection file with related anime AND series sharing the same tvdbid ########################
   def anidbCollectionMapping(self, metadata, anime, anidbid_table=[]):
     global AniDB_collection_tree, SERIE_LANGUAGE_PRIORITY
-    related_anime_list = []; metadata_id_source, metadata_id_number = metadata.id.split('-', 1)
+    metadata_id_source, metadata_id_number, related_anime_list = metadata.id.split('-', 1), []
```

Comment:
This will not work. Python will complain that the split is not returning 3 pieces.
EX:
```
>>> qwe = "asd-123"
>>> qwe.split('-',1)
['asd', '123']
>>> z, x, c = qwe.split('-',1), []
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: not enough values to unpack (expected 3, got 2)
```